RubberduckTests/IntegrationTests/FakeTests.bas

```diff
@@ -238,3 +238,100 @@ TestExit:
 TestFail:
     Assert.Fail "Test raised an error: #" & Err.Number & " - " & Err.Description
 End Sub
+
+'@TestMethod
+Public Sub NowFakeWorks()
+    On Error GoTo TestFail
+
+    With Fakes.Now
+        .Returns #1/1/2018 9:00:00 AM#
+        Assert.IsTrue Now = #1/1/2018 9:00:00 AM#
+        .Verify.Once
+    End With
+
+TestExit:
+    Exit Sub
+TestFail:
+    Assert.Fail "Test raised an error: #" & Err.Number & " - " & Err.Description
+End Sub
+
+'@TestMethod
+Public Sub NowFakePassThroughWorks()
+    On Error GoTo TestFail
+
+    With Fakes.Now
+        .Returns #1/1/2018 9:00:00 AM#
+        .Passthrough = True
+        Assert.IsTrue Now <> #1/1/2018 9:00:00 AM#
+        .Verify.Once
+    End With
+
+TestExit:
+    Exit Sub
+TestFail:
+    Assert.Fail "Test raised an error: #" & Err.Number & " - " & Err.Description
+End Sub
+
+'@TestMethod
+Public Sub TimeFakeWorks()
+    On Error GoTo TestFail
+
+    With Fakes.Time
+        .Returns #9:00:00 AM#
+        Assert.IsTrue Time = #9:00:00 AM#
+        .Verify.Once
+    End With
+TestExit:
+    Exit Sub
+TestFail:
+    Assert.Fail "Test raised an error: #" & Err.Number & " - " & Err.Description
+End Sub
+
+'@TestMethod
+Public Sub TimeFakePassThroughWorks()
+    On Error GoTo TestFail
+
+    With Fakes.Time
+        .Returns #9:00:00 AM#
+        .Passthrough = True
+        Assert.IsTrue Time <> #9:00:00 AM#
```

Comment:
this unit-test only works if it's not actuall 9AM. that's probably fine, though, soo... not much to do there